### PR TITLE
Converting "member" to "type(member)" on line 311 of completion.py to fix issue 272

### PR DESCRIPTION
**Pull request by bradleydamato**
Passes: 
`completion_test.py`
`core_test.py`
`main_test.py`

### CODE REVIEW

**Comment by bradleydamato** [comment]
@dbieber here is the PR for Issue 272 https://github.com/google/python-fire/issues/272

File: fire/completion.py [hunk 1]
```diff
@@ -308,7 +308,7 @@ def MemberVisible(component, name, member, class_attrs=None, verbose=False):
     return False
   if verbose:
     return True
-  if member in (absolute_import, division, print_function):
+  if type(member) in (absolute_import, division, print_function):
```
Comment (dbieber) [inline fire/completion.py:311]: I don't think this is quite the check we want.

In Python 3 type(absolute_import) is __future__._Feature, which is fine.
However in Python 2 type(absolute_import) is instance which is too broad.

Instead we can do `if (member is absolute_import or member is division or member is print_function)`

Comment (bradleydamato) [reply fire/completion.py:311]: Sounds good @dbieber I'll make the change and push again

**Comment by bradleydamato** [comment]
Looks like the line is too long; third time is the charm :). BRB.

**Comment by dbieber** [comment]
Looks good, thanks for the PR.

**Comment by dbieber** [comment]
Merged in c3789064e2e161992e3c5a23eb0c47b66072707f

**Pull request closed without merging**